### PR TITLE
ListItem/Alphi Added font to assets; Edited Shadow; Added new Color; Combined with new code.

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

**Comment by aahenry51** [comment]
This is the build-in font.

oldfont

This is the Poppins font.

Newfont

**Review by Nap1400 — APPROVED** [review]
I need this font for the StepsProgress component.

**Comment by aahenry51** [comment]
Shadow

Integrated my code, Changed shadow properties for list item, Changed Border radius, Added a new color.

**Review by carizzuti — APPROVED** [review]
Color looks good and UI is clean

File: src/components/listItem/ListItem.js [hunk 1]
```diff
@@ -1,48 +1,45 @@
...
   title,
   subTitle,
-  titleContainerStyle,
-  subTitleContainerStyle,
-  listItemContainerStyle,
-  titleStyle,
-  subTitleStyle,
-  iconContainerStyle,
-  ...rest
+  MaxTitleLength,
```
Comment (raynesmike) [inline src/components/listItem/ListItem.js:10]: camelCase this one

**Comment by lasharela** [comment]
image
Subtitle text doesn't match . It's regular in figma

File: src/components/listItem/ListItem.js (hunk 1 above)
Comment (aahenry51) [reply src/components/listItem/ListItem.js]: Fixed

**Comment by aahenry51** [comment]
> image
> 
> Subtitle text doesn't match . It's regular in figm
Both the title and subtitle are regular in figma.

**Review by leo-he-1 — APPROVED** [review]
I get errors when I run this branch. Not sure if it's a problem on my end.
image

**Review by raynesmike — APPROVED** [review]
I approve with all these changes

**Comment by aahenry51** [comment]
image

**Comment by aahenry51** [comment]
image

**Pull request merged**